src/routes/data-decode/entities/data-decoded.entity.ts

```diff
@@ -6,9 +6,16 @@ export class DataDecoded {
   method: string;
   @ApiPropertyOptional({ type: [DataDecodedParameter], nullable: true })
   parameters: DataDecodedParameter[] | null;
+  @ApiPropertyOptional({ type: String, nullable: true })
+  humanDescription?: string;
```

Comment:
I've had it as null before but had to add the property to a lot of mappers, entities and tests. If we don't mind a larger diff from this PR I can change it back to null.

Reply:
We should also not expose the property if the feature is not enabled.

(For the tests, by using the builders we can cover most of the default)

Reply:
I am not sure how we can only expose the `humanDescription` property from the class if the feature is enabled. Do you have an example of this?

Reply:
> I am not sure how we can only expose the `humanDescription` property from the class if the feature is enabled. Do you have an example of this?

This is the first time that the CGW has a feature rollout with NestJS so we don't have other examples. I can explore different alternatives on how this can be made.